.pre-commit-config.yaml

```diff
@@ -12,6 +12,6 @@ repos:
     rev: 6.1.0
     hooks:
       - id: flake8
-        language_version: python3.11
+        language_version: python3.12
 default_language_version:
-  python: python3.11
+  python: python3.12
```

Comment:
bumping since the repo `.python-version` is 3.12.2

Reply:
Have you tested the script still works ?

Reply:
yes, still works:
```
% topicctl apply examples/local-cluster/topics/topic-default2.yaml --dry-run --json-output | py/parse_and_notify.py
```
produced
https://app.datadoghq.com/event/explorer?query=source%3Atopicctl%20sentry_region%3Aben_test%20&cols=&event=AgAAAZCjH76Idce8hwAAAAAAAAAYAAAAAEFaQ2pINzZJQUFBZk9aNXJlLUZucjQ2agAAACQAAAAAMDE5MGEzMWYtZDllMC00ZDIwLTgwZDItYWY1NzNhNGIyNGU5&messageDisplay=expanded-lg&options=&refresh_mode=sliding&sort=DESC&view=spans&from_ts=1720722860311&to_ts=1720723760311&live=true